src/shared/components/rangeSelector/RangeSelector.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { observer} from "mobx-react";
+import {
+    VictoryChart, VictoryAxis, VictoryLabel, VictoryBrushContainer
+} from 'victory';
+import CBIOPORTAL_VICTORY_THEME, {baseLabelStyles} from "../../../shared/theme/cBioPoralTheme";
+import { observable } from 'mobx';
+import autobind from 'autobind-decorator';
+
+interface IRangeSelectorProps
+{
+    initialRange: {min: number, max: number}
+    maxAxisValue: number;
+    styleOpts: any; // see victory styles, or styleOptsDefaultProps in SurvivalChart for examples
+    xAxisTickCount: number
+    selectedDomain: any;
+    minAxisValue?: number;
+    updateDomain?: (domain: any)=>void;
+}
+
+@observer
+export default class RangeSelector extends React.Component<IRangeSelectorProps> {
+    constructor(props:IRangeSelectorProps) {
+        super(props);
+    }
+
+    public static defaultProps = {
+        minAxisValue: 0
+    };
+
+    @autobind
+    private onSelection(domain: any, props: any) {
```

Comment:
@alisman just curious, should we not bind the event at all if the property is not available?